rllib/connectors/tests/test_agent.py

```diff
@@ -470,11 +469,20 @@ def test_connector_pipline_with_view_requirement(self):
         )
 
         env = gym.make("CartPole-v1")
-        policy = PPOTorchPolicy(
```

Comment:
This policy is not compatible with RLModule, so I needed to get it from algorithm based on the flag.